docs/setup-dev-environment.md

```diff
@@ -162,11 +172,55 @@ local repository, so that it can read from the `package.json` file.
 
 `cd ./colouring-london/app && npm install`
 
+## Load OpenStreetMap test polygons
```

Comment:
Hi @matkoniecz @tomalrussell - I have got the application running using the changes in this PR, but currently without data loading. I think it makes sense to add in the OSM test polygons instructions here (I will modify the other docs to avoid duplication and link everything from the main README too).

Can you spot what, if anything I may have done wrong here and advise how my version of the site should look with these test buildings? Are they only in a small part of the city?

![Screenshot 2022-02-03 at 15 26 34](https://user-images.githubusercontent.com/5486164/152373362-de3e4b37-787a-4a77-800a-712c08c603c0.png)

Reply:
https://github.com/colouring-cities/colouring-london/blob/master/etl/get_test_polygons.py#L21

Buildings should be around https://www.openstreetmap.org/#map=19/51.52450/-0.13387

Reply:
ok thanks, I seem to have 1779 buildings in the db, but they don't show up on the map so unsure why

Reply:
On `npm start` I've added the environment variable: `TILECACHE_PATH=../maintenance/tiles/` <- this doesn't seem to have worked either, did I do this wrong maybe?

Reply:
also @matkoniecz do you know the answer to this question^

Reply:
Doesn't look like files are being created in the `tilecache` dir.
Error that repeats in the console when running the node app:

```
Postgis Plugin: authentication method 10 not supported
Connection string: 'host=localhost port=5432 dbname=testlondon user=cldbadmin connect_timeout=4 fallback_application_name=mapnik'

    at renderDataSourceTile (/home/ed/colouring-london/app/colouring-london-app/src/tiles/renderers/renderDataSourceTile.ts:34:1)
    at renderBuildingTile (/home/ed/colouring-london/app/build/webpack:/colouring-london-app/src/tiles/rendererDefinition.ts:47:1)
    at stitchOrRenderBuildingTile (/home/ed/colouring-london/app/build/webpack:/colouring-london-app/src/tiles/rendererDefinition.ts:57:1)
    at getTileWithCaching (/home/ed/colouring-london/app/build/webpack:/colouring-london-app/src/tiles/renderers/getTileWithCaching.ts:7:1)
    at /home/ed/colouring-london/app/build/webpack:/colouring-london-app/src/tiles/tileserver.ts:20:1
```

The suggestion I've found online is that `libpq`needs to be updated, however, when I try installing the following I get:

```
libpq-dev is already the newest version (14.2-1.pgdg18.04+1).
```

&

```
libpq5 is already the newest version (14.2-1.pgdg18.04+1).
```

Reply:
It is likely that `node-mapnik` was build with a different `libpq`. It looks like this issue is reported at https://github.com/mapnik/mapnik/issues/4283 

Can you try the workaround of setting postgres to store passwords with `md5`? [Postgres docs](https://www.postgresql.org/docs/14/auth-password.html)

(Otherwise could look into building node-mapnik - see for example slightly old notes in https://github.com/mapnik/mapnik-support/issues/110)

Reply:
Ok, for now I have it working with Postgres-12, I can look at that issue properly at a later date when working on #531